docs/source/testing/tox.rst

```diff
@@ -3,20 +3,90 @@
 ``tox``
 =======
 
+``tox`` is an automation project we use to run our testing scenarios. It gives us
+the ability to create a dynamic matrix of many testing scenarios, isolated testing environments
+and a provides a single entry point to run all tests in an automated and repeatable fashion.
+
+Documentation for tox can be found `here<https://tox.readthedocs.io/en/latest/>`_.
+
 
 .. _tox_environment_variables:
 
 Environment variables
 ---------------------
 
+When running ``tox`` we've allowed for the usage of environment variables to tweak certain settings
+of the playbook run using ``--extra-vars``. This is mostly useful for usage in Jenkins jobs or for using
```

Comment:
"usage", "using", and "useful" can be trimmed down (appears too often in this one paragraph).

    It's helpful in Jenkins jobs or for manual test runs of ceph-ansible

Instead of:

    This is mostly useful for usage in Jenkins jobs or for using [...]